platform-includes/enriching-events/add-attachment/python.mdx

```diff
@@ -1,28 +1,22 @@
-Attachments live on the `Scope` and will be sent with all events.
+Attachments live on the [`Scope`](/platforms/python/enriching-events/scopes/). The SDK sends all attachments on a given `Scope` with every non-transaction event captured within the `Scope`. Optionally, an attachment can be sent with all events, including transactions, when the attachment is added with the `add_to_transactions` option set to `True`.
 
 ```python
 # Add an attachment
-from sentry_sdk import configure_scope
+from sentry_sdk import Scope
 
-with configure_scope() as scope:
-    scope.add_attachment(bytes=b"Hello World!", filename="attachment.txt")
-    scope.add_attachment(path="/path/to/attachment/file.txt")
-```
-
-An attachment has the following fields:
-
-`path`
+# Scope.get_isolation_scope() or Scope.get_global_scope() can also be used.
+# Read about the different scopes on our Scope docs page.
+scope = Scope.get_current_scope()
 
-The path to the attachment file. The `filename` and `content_type` will be inferred if not explicitly provided when using this mode.
+# Add attachment titled "attachment.txt" with content "Hello World!"
+scope.add_attachment(bytes=b"Hello World!", filename="attachment.txt")
 
-`bytes`
+# Attach the file located at /path/to/attachment/file.txt
+scope.add_attachment(path="/path/to/attachment/file.txt")
 
-The content of the attachment as bytes.
-
-`filename`
-
-The filename is required if using `bytes` and will be displayed in [sentry.io](https://sentry.io).
+# This attachment will also be sent with transactions
+scope.add_attachment(path="/other/attachment.txt", add_to_transactions=True)
+```
 
-`content_type`
 
-The type of content stored in this attachment. Any [MIME type](https://www.iana.org/assignments/media-types/media-types.xhtml) may be used; the default is `application/octet-stream`.
+Our [API Docs](https://getsentry.github.io/sentry-python/apidocs.html#sentry_sdk.Scope.add_attachment) describe the `add_attachment` method's parameters in detail.
```

Comment:
we've in the past kept these docs isolated from apidocs (no other references to apidocs in `sentry-docs`) and not really advertized them so I wouldn't start now.
Can you keep the original parameter descriptions here and add `add_to_transactions`? You can keep the examples you added above, they're useful.

Reply:
> we've in the past kept these docs isolated from apidocs (no other references to apidocs in sentry-docs) and not really advertized them so I wouldn't start now.

Huh, I did not know that. There is actually [one other reference I know of in the apidocs](https://docs.sentry.io/platforms/python/tracing/instrumentation/custom-instrumentation/#add-a-transaction) (which I added since I did not know that we did not want to advertise this).

Why do we not want to link to the API Docs? Imo it is redundant to have the same information (in this case, the parameters) documented in multiple places, and it increases maintenance burden, since any changes need to be documented in two places. 

And if we never really advertise that the API Docs exist, why do we even maintain them? I think we should either reference those docs or delete them.

Reply:
because nobody maintained the apidocs and they are wayyy outdated.

Reply:
I showed them some love some time ago. But stopped because wanted to wait until the Hub is gone to really revamp them (and add versioned docs) 
I still think having versioned api docs is the way to go, 
but it is also true, that in the current state they are not useful at all